slam-ws/src/transform/launch/setup_tf.launch

```diff
@@ -1,6 +1,6 @@
 <launch>
     
-    <arg name="robot_model" defaul="husky_gazebo"/>
```

Comment:
Merge the current `dev` onto your branch and this fix will already be implemented